### PR TITLE
Prevent importing db core from db runtime

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- Importing things in `src/core` from `src/runtime` will cause production to include rollup, vite and other unwanted things.
- This eslint rule will error if you do this. 

## Testing

- Verified locally

## Docs

N/A, bug fix

### CODE REVIEW

File: packages/db/src/core/errors.ts [hunk 1]
```diff
@@ -37,46 +37,14 @@ export const SHELL_QUERY_MISSING_ERROR = `${red(
...
 
-export const SEED_DEFAULT_EXPORT_ERROR = (fileName: string) => {
-	return SEED_ERROR(`Missing default function export in ${bold(fileName)}`);
-};
-
 export const EXEC_DEFAULT_EXPORT_ERROR = (fileName: string) => {
 	return EXEC_ERROR(`Missing default function export in ${bold(fileName)}`);
 };
 
-export const REFERENCE_DNE_ERROR = (columnName: string) => {
```
Comment (bholmesdev) [inline packages/db/src/core/errors.ts:56]: Nit: maybe we can move `errors.ts` to the base of the `src` directory instead of splitting into two modules?

**Review by bholmesdev — COMMENTED** [review]
Excel-lint change! Left a comment on organizing our errors

File: packages/db/src/core/errors.ts (hunk 1 above)
Comment (matthewp) [reply packages/db/src/core/errors.ts:56]: We definitely can, it's interesting though that none of the errors are shared between. in which case, why have them together?

Comment (bholmesdev) [reply packages/db/src/core/errors.ts:56]: Huh, honestly expected at least _one_ to be shared. Guess we can keep separate if that's the case!

**Review by bholmesdev — APPROVED** [review]

**Review by ematipico — APPROVED** [review]

**Pull request merged**